src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
Also we're thinking of maintaining a fork in our organization's repo since we use it already in a few places